src/features/saksoversikt/utils.tsx

```diff
@@ -52,7 +52,7 @@ export const statusIcon = (status: VilkårVurderingStatus) => {
     }
 };
 
-type Vilkårsinformasjon = {
+export type Vilkårsinformasjon = {
```

Comment:
Kanskje endre denne til 
```suggestion
export interface Vilkårsinformasjon {
```
med det samme.